test/functional/install/install-multiple.bats

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -n test-bundle -f /file_1,/file_2 "$TEST_NAME"
+	sudo mkdir -p "$TARGETDIR"/usr/share/clear/bundles
+	sudo touch "$TARGETDIR"/usr/share/clear/bundles/test-bundle
```

Comment:
same here, every bundle that is created with the test library includes the tracking file by default:
https://github.com/clearlinux/swupd-client/blob/master/test/functional/testlib.bash#L1585